configure/CONFIG_SITE

```diff
@@ -13,23 +13,20 @@
 #######################################################################
 
 # Use Perl compatible regular expressions (PCRE) instead of basic regex
-#USE_PCRE=YES
-
-# For PCRE on Linux
-#    requires package pcre-devel (or libpcre3-dev) for compiling
-#    requires package pcre (or libpcre3) for running the Gateway
+#USE_PCRE=2
 
+# For PCRE on Linux make sure PCRE 1 or 2 deveopment packeges are installed.
 ifeq ($(OS_CLASS),WIN32)
 
-    # For PCRE on Windows
+    # For PCRE 1 on Windows
     #    install libraries and includes from
     #    https://sourceforge.net/projects/gnuwin32/files/pcre/7.0/pcre-7.0.exe/download
     #    and set the install location here
     PCRE_DIR=C:/Pcre
 
     # For REGEX on Windows
     #    compile gnuregex inside an EPICS support module from the sources at
-    #    http://www.aps.anl.gov/epics/download/extensions/gnuregex0_13.tar.gz
+    #    https://www.aps.anl.gov/epics/download/extensions/gnuregex0_13.tar.gz
```

Comment:
Since you're changing this, please make it `https://epics.anl.gov/download/extensions/gnuregex0_13.tar.gz`